docs/feature_haptic_feedback.md

```diff
@@ -42,14 +42,21 @@ First you will need a build a circuit to drive the solenoid through a mosfet as
 [Wiring diagram provided by Adafruit](https://playground.arduino.cc/uploads/Learning/solenoid_driver.pdf)
 
 
-| Settings                 | Default       | Description                                           |
-|--------------------------|---------------|-------------------------------------------------------|
-|`SOLENOID_PIN`            | *Not defined* |Configures the pin that the Solenoid is connected to.  |
-|`SOLENOID_DEFAULT_DWELL`  | `12` ms       |Configures the default dwell time for the solenoid.    |
-|`SOLENOID_MIN_DWELL`      | `4` ms        |Sets the lower limit for the dwell.                    |
-|`SOLENOID_MAX_DWELL`      | `100` ms      |Sets the upper limit for the dwell.                    |
-
-?> Dwell time is how long the "plunger" stays activated.  The dwell time changes how the solenoid sounds.
+| Settings                   | Default              | Description                                           |
+|----------------------------|----------------------|-------------------------------------------------------|
+|`SOLENOID_PIN`              | *Not defined*        |Configures the pin that the Solenoid is connected to.  |
+|`SOLENOID_DEFAULT_DWELL`    | `12` ms              |Configures the default dwell time for the solenoid.    |
+|`SOLENOID_MIN_DWELL`        | `4` ms               |Sets the lower limit for the dwell.                    |
+|`SOLENOID_MAX_DWELL`        | `100` ms             |Sets the upper limit for the dwell.                    |
+|`SOLENOID_DWELL_STEP_SIZE`  | `1` ms               |The step size to use when `HPT_DWL*` keycodes are sent |
+|`SOLENOID_DEFAULT_BUZZ`     | `0` (disabled)       |On HPT_RST buzz is set "on" if this is "1"             |
```

Comment:
I'm not sure if I like the name for this, but I'm not sure if there is a better way to name this.

Reply:
Well BUZZ is used in other places such as the HPT_BUZ keycode "Toggle solenoid buzz on/off".
If we were to rename this, then we would need to rename it anywhere.
If Buzz is not a good name, than maybe rattle?
keycode would become HPT_RAT. macro would be SOLENOID_DEFAULT_RATTLE?